src/test/resources/data/FeedbackSessionResultsTest.json

```diff
@@ -179,9 +179,9 @@
       "googleId": "FSRTest.student1InCourse1",
       "email": "FSRTest.student1InCourse1@gmail.tmt",
       "course": "FSRTest.idOfTypicalCourse1",
-      "name": "student1 In Course1</td></div>'\"",
+      "name": "student1 In Course1",
       "comments": "comment for student1InCourse1",
-      "team": "Team 1.1</td></div>'\"",
```

Comment:
Why do we need the change this?

Reply:
I thought the HTML tags were just typos that never got fixed?

Reply:
Nope. There are there for some reasons:
- Text sanitization
- HTML code escape